fastchat/conversation.py

```diff
@@ -480,6 +480,16 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# OpenChat 3.5 default template
+register_conv_template(
+    Conversation(
+        name="openchat_3.5",
+        roles=("GPT4 Correct User", "GPT4 Correct Assistant"),
+        sep_style=SeparatorStyle.FALCON_CHAT,
+        sep="<|end_of_turn|>",
```

Comment:
Just curious, why do you choose this template?

Reply:
Because the conversation template looks like https://huggingface.co/openchat/openchat_3.5/discussions/5#6544c046ee8e348cf9990a8b

If there is no system prompt, there should be no extra separators, FALCON_CHAT will do the job.

Reply:
@imoneoi Why do you use "GPT4 Correct User"/"GPT4 Correct Assistant" instead of simple "User"/"Assistant"?